src/views/ConflictResolution/ConflictMap/styles.scss

```diff
@@ -0,0 +1,37 @@
+.conflict-map {
+    position: relative;
+    border: 1px solid var(--color-separator);
+    border-bottom: 2px solid var(--color-separator);
+    border-radius: 4px;
+
+    &.conflicted {
+        border-bottom: 2px solid var(--color-danger);
+        background-color: var(--color-foreground);
+    }
+
+    &.selected {
+        border-bottom: 2px solid var(--color-accent);
+        background-color: var(--color-foreground);
+    }
+
+    .map {
+        height: 300px;
+
+    }
+
+    .layer-switcher {
+        position: absolute;
+        bottom: var(--spacing-small);
+        left: var(--spacing-small);
+        z-index: 11;
```

Comment:
Let's use 1 if possible and especially for z-index let's use variables from a single place so that it will be easier to maintain hierarchy if there are multiple-use at the same time.